tests/test_servers/test_http_server.py

```diff
@@ -1,29 +1,50 @@
 import json
+import os
+import subprocess
 import tempfile
 import unittest
-
 from pathlib import Path
 
 import pytest
 
 import runhouse as rh
+
+from runhouse.globals import rns_client
 from runhouse.servers.http.http_utils import b64_unpickle, pickle_b64
 
 from tests.test_resources.test_clusters.conftest import (
+    local_docker_cluster_public_key_den_auth,
     local_docker_cluster_public_key_logged_in,
 )
 
-from tests.test_servers.conftest import summer
+from tests.test_servers.conftest import local_client, local_client_with_den_auth, summer
 
+INVALID_HEADERS = {"Authorization": "Bearer InvalidToken"}
 
-@pytest.mark.usefixtures("cluster")
-class TestHTTPServer:
-    """Start HTTP server in a docker container running locally"""
-
-    UNIT = {"cluster": [local_docker_cluster_public_key_logged_in]}
-    LOCAL = {"cluster": [local_docker_cluster_public_key_logged_in]}
-    # TODO add local_docker_cluster_passwd into LOCAL too?
 
+@pytest.mark.usefixtures("cluster")
+@pytest.mark.den_auth
+class TestHTTPServerDocker:
+    """
+    Test the HTTP server running on a local Docker container.
+
+    These will be tested on a container with Den Auth enabled, and a container without.
+    """
```

Comment:
We can still run these with MINIMAL though to run on remote instances though, correct?